### DISCUSSION TITLE
Updated Russian translation for 0.47

### DISCUSSION THREAD

**Discussion by SanokKule** [rejetto/hfs#312] [General]
In case there are more strings to be updated, leave this open until 0.47 is out i guess.
Love the server ❤️
[hfs-lang-ru v1.98.zip](https://github.com/rejetto/hfs/files/12114729/hfs-lang-ru.v1.98.zip)

**Comment by rejetto** [comment]
you got a couple of new entries for 0.47.0

**Reply by SanokKule** [reply]
Oh i didn't see the 'next' branch, give me a moment

**Reply by SanokKule** [reply]
[hfs-lang-ru v1.99.zip](https://github.com/rejetto/hfs/files/12251934/hfs-lang-ru.v1.99.zip)

**Comment by rejetto** [comment]
> Oh i didn't see the 'next' branch, give me a moment

yeah, sorry :)

**Comment by SanokKule** [comment]
[hfs-lang-ru v2.0.zip](https://github.com/rejetto/hfs/files/12262813/hfs-lang-ru.v2.0.zip)

**Reply by rejetto** [reply]
forgot to include it in beta7
